### PR TITLE
LRAC-13060 remove expando column association for new wizard

### PR DESCRIPTION
**Pull request by thiago-buarqque**
Related to [LRAC-13060](https://issues.liferay.com/browse/LRAC-13060)

### CODE REVIEW

**Comment by thiago-buarqque** [comment]
ci:test:sf

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Review by marcosapmf — CHANGES_REQUESTED** [review]

File: modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/model/listener/ExpandoColumnModelListener.java [hunk 1]
```diff
@@ -40,6 +45,48 @@
...
+					).<String, Serializable>put(
+						"syncedUserFieldNames", syncedUserFieldNames2
+					).build());
+			}
+			catch (Exception exception) {
+				_log.error(exception);
+			}
+		}
+		else {
+			analyticsAssociationLocalService.deleteAnalyticsAssociations(
```
[the remark this answers is not in the document]
Comment (thiago-buarqque) [reply modules/apps/analytics/analytics-settings-impl/src/main/java/com/liferay/analytics/settings/internal/model/listener/ExpandoColumnModelListener.java]: Hey @marcosapmf. Just made the change, thanks.

**Comment by thiago-buarqque** [comment]
ci:reevaluate:1267401_8920

**Comment by ViniLopes20** [comment]
ci:reevaluate:1267401_8920

**Comment by thiago-buarqque** [comment]
ci:test:relevant

**Review by marcosapmf — APPROVED** [review]

**Comment by thiago-buarqque** [comment]
Follow up fix sent to to Brian [here](https://github.com/brianchandotcom/liferay-portal/pull/134244)

**Pull request closed without merging**